### PR TITLE
Adding list of courses in Uzbek language

### PR DESCRIPTION
**Pull request by rusyasoft**
Adding list of courses in Uzbek language that could be very useful to some Uzbek students. Also additional python book reference in Uzbek language

## What does this PR do?
Adding list of courses in Uzbek language that could be very useful to some Uzbek students. Also additional python book reference in Uzbek language

## Checklist:
- [*] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [*] Search for duplicates.
- [*] Include author(s) and platform where appropriate.
- [*] Put lists in alphabetical order, correct spacing.
- [*] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-uz.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+### Mundarija
+
+* [Python](#python)
+
+
+### Python
+
+* [Python Dasturlash tili va uning imkoniyatlari bo'yicha uslubiy qo'llanma](http://ru.diveintopython.net/toc.html) - Buxoro Davlat Universisteti Fizika-matematika fakultetida chiqarilgan. Toshboyeva Guljahon O'tkir qizi
```
Comment (eshellman) [inline books/free-programming-books-uz.md:8]: dead link

**Comment by eshellman** [comment]
the tutorials.uz links resolve to youtube and telegram links are there youtube playlists to use instead. we have dificulties with this type of link similar to those with link shorteners.

File: books/free-programming-books-uz.md (hunk 1 above)
Comment (rusyasoft) [reply books/free-programming-books-uz.md:8]: that was my mistake fixing the link

**Comment by rusyasoft** [comment]
> r

if you open the link directly it asks you to join the telegram group bu the links I'm providing are referencing to the website where you get well organized index based youtube videos in Uzbek language

**Comment by rusyasoft** [comment]
> the tutorials.uz links resolve to youtube and telegram links are there youtube playlists to use instead. we have dificulties with this type of link similar to those with link shorteners.

if you open the link directly it asks you to join the telegram group bu the links I'm providing are referencing to the website where you get well organized index based youtube videos in Uzbek language

**Comment by eshellman** [comment]
feel free to reopen if you find non-redirector links.

**Pull request closed without merging**